gluten-core/src/main/scala/io/glutenproject/execution/JoinUtils.scala

```diff
@@ -259,7 +260,8 @@ object JoinUtils {
         else getDirectJoinOutputSeq(joinType, streamedOutput, buildOutput),
         validation),
       substraitContext,
-      operatorId)
+      operatorId,
+      inputBuildOutput.size + inputBuildOutput.size)
```

Comment:
Do you intend to use `build out + stream out`?

Reply:
Thanks, just fixed.